src/utils/setup/create-podcast-pages.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+import { getTagPageUriComponent } from '../get-tag-page-uri-component';
+import { fetchBuildTimeMedia } from '../../utils/setup/fetch-build-time-media';
+import { Podcast } from '../../interfaces/podcast';
+
+export const createPodcastPages = async (
+    createPage: Function,
+    allPodcasts: Podcast[],
+    metadata: object
+) => {
+    allPodcasts.map((podcast: Podcast) => {
+        const months = ["Jan", "Feb", "Mar","Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+        const publishDate = new Date(podcast.publishDate);
+        const slug = `/podcasts/${getTagPageUriComponent(podcast.title)}`;
+        podcast.slug = slug;
+        podcast.publishDate = months[publishDate.getMonth()+1] + " " + publishDate.getDate() + ", " + publishDate.getFullYear();
```

Comment:
I think the +1 is off here, I logged the before/after and noticed it seems off by a month. For example:

```
Before: Tue, 25 Feb 2020 13:00:00 +0000
After: Mar 25, 2020
```

Reply:
actually its -1 , to map for the corresponding index in month array. I wish there is inbuilt method in date library , but there isn't one.